kohttp/src/main/kotlin/io/github/rybalkinsd/kohttp/interceptors/RetryInterceptor.kt

```diff
@@ -52,6 +59,14 @@ class RetryInterceptor(
 
     private fun shouldDelay(attemptsCount: Int) = invocationTimeout > 0 && attemptsCount > 0
 
-    internal fun isRetry(response: Response, attemptsCount: Int): Boolean =
-            attemptsCount < failureThreshold && response.code() in errorStatuses
+    internal fun calculateNextRetry(response: Response, attemptsCount: Int, delay: Long, maxDelayTime: Int): Long? {
+        val retryAfter = response.header("Retry-After")?.toLongOrNull()
```

Comment:
probably need an ext function to get header by name.

BTW, did you check what is the metric of `Retry-After` values? s/ms ?

Reply:
> probably need an ext function to get header by name.

OK!

> BTW, did you check what is the metric of Retry-After values? s/ms ?

The unit of Retry-After value is seconds. But in my code it is millisecond.
Sorry, I'll fix it.